csharp/test/Ice/operations/TwowaysAMI.cs

```diff
@@ -1344,6 +1345,50 @@ internal static void Run(TestHelper helper, IMyClassPrx p)
                         TestHelper.Assert(p2.DictionaryEqual(p1) && ReturnValue.DictionaryEqual(p1));
                     }
                 }).Wait();
+
+            {
+                void CompareStreams(System.IO.Stream s1, System.IO.Stream s2)
+                {
+                    int v1, v2;
+                    do
+                    {
+                        v1 = s1.ReadByte();
+                        v2 = s2.ReadByte();
+                        TestHelper.Assert(v1 == v2);
+                    }
+                    while (v1 != -1 && v2 != -1);
+
+                    s1.Dispose();
+                    s2.Dispose();
+                }
+
+                if (p.Protocol != Protocol.Ice1)
+                {
+                    Task.Run(async () =>
+                    {
+                        await p.OpSendStream1Async(File.OpenRead("AllTests.cs")).ConfigureAwait(false);
+
+                        await p.OpSendStream2Async("AllTests.cs", File.OpenRead("AllTests.cs")).ConfigureAwait(false);
+
+                        System.IO.Stream stream;
+                        string fileName;
+
+                        stream = await p.OpGetStream1Async().ConfigureAwait(false);
```

Comment:
Instead of this Dispose in CompareStreams, `using` could be cleaner, as in:
```
{
    using var resultStream = await ...
    using var fileStream = File.OpenRead(...)
    AssertStreamEqual(resultStream, fileStream);
}
```

```